### PR TITLE
Merging changes up to the end of lesson 9

### PR DESCRIPTION
**Pull request by hclivess**
Merging changes up to the end of lesson 9

### CODE REVIEW

File: transcripts/06-methods/4.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@
 1:31 oh look, in the subsequent sections on how Python deals with this,
 1:36 because this kind of flexibility is super powerful and Python does support it
 1:40 it just doesn't support it in the traditional method overloading way
-1:44 that you might be familiar with coming from C++, Java, C# and so on.
+1:44 that you might be familiar with coming from C++, Java, CSharp and so on.
```
Comment (mikeckennedy) [inline transcripts/06-methods/4.txt]: @hclivess Actually C# is better here.

Comment (hclivess) [reply transcripts/06-methods/4.txt]: OK, I'll change it back to C#, but I noticed that the application which shows the captions has issues with some characters, sometimes it does not display "++" or "<" and ">" symbols

Comment (mikeckennedy) [reply transcripts/06-methods/4.txt]: Ok, weird. Let's see how it comes out. If doesn't render correctly we should definitely switch this out. :)

**Comment by mikeckennedy** [comment]
@hclivess Excellent work. Does this conclude this course?

**Comment by hclivess** [comment]
@mikeckennedy I think it does, Mike

**Comment by mikeckennedy** [comment]
Perfect, I'll merge it all back into the site. Thanks!

**Comment by mikeckennedy** [comment]
All the changes are now pushed to https://training.talkpython.fm and should be live now.

**Pull request merged**